documentation/RLC-Swagger-quickstart.md

```diff
@@ -0,0 +1,309 @@
+Getting Started - Generate the RLC rest-level client libraries with Swagger
+===========================================================================
+
+# Before you start
+
+Please refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. We highly recommand to read [this blog](https://devblogs.microsoft.com/azure-sdk/azure-rest-libraries-for-javascript/) to get familiar with REST libraries for JavaScript. 
+
+:warning: Note: if you’re still generating from Swagger with RLC, please read [this doc](./RLC-Swagger-quickstart.md) for Swagger specific details.
```

Comment:
Why are you referring to this doc itself ?

Reply:
Remove it